drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/SubHeaderBuilder.java

```diff
@@ -122,7 +185,11 @@ private void makeInsert(final ActionInsertFactCol52 column) {
             makeHeaderAndTitle(ACTION,
                                "");
 
-            fieldRow.createCell(columnIndex).setCellValue(column.getFactType() + " " + column.getBoundName() + " = new " + column.getFactType() + "(); insert( " + column.getBoundName() + " );");
+            fieldRow.createCell(columnIndex).setCellValue(String.format("%s %s = new %s(); insert( %s );",
```

Comment:
Maybe also here we could use `MessageFormat` as we pass twice `column.getBoundName()`